tests/service/views/test_exceptions.py

```diff
@@ -99,6 +99,7 @@ def test_content_type_headers_exc(svc_client_with_repo):
 
 @pytest.mark.service
 @pytest.mark.integration
+@pytest.mark.remote_repo("old")
```

Comment:
We should add `remote_repo` to the list of valid markers in https://github.com/SwissDataScienceCenter/renku-python/blob/35f9d22f571b53df0ac5f52ded5ebabd828b68d5/pyproject.toml#L249